src/components/miniplayer/View.tsx

```
@@ -0,0 +1,131 @@
+  };
+  const collapse = () => {
+    'worklet';
+    miniplayerHeight.value = withTiming(MinPlayerHeight, { duration: 500 });
+    artworkOpacity.value = withTiming(1);
+    runOnJS(setLrcVisible)(false);
+  };
+  const onArtworkPress = () => {
+    if (artworkOpacity.value === 1) {
+      return (artworkOpacity.value = withTiming(0, { duration: 100 }, () => {
+        runOnJS(setLrcVisible)(true);
+      }));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)